lib/external.py

```diff
@@ -215,3 +217,31 @@ def check_django(self, machine, manage_py, exception=True):
 
             return ret == 0 # 0 - no issues, 1 - problems
 
+    @contextmanager
+    def connect_rds(self):
+        """
+        Context manager with established connection to rds
```

Comment:
Say which RDS and also mention loading credentials from Vault (which adds an extra tunnel overhead)